tests/bridge_emit_tests.cpp

```diff
@@ -0,0 +1,295 @@
+
+  // Create destination account
+  create_accounts({"rec1"_n});
+  set_code("rec1"_n, testing::contracts::evm_bridge_receiver_wasm());
+  set_abi("rec1"_n, testing::contracts::evm_bridge_receiver_abi().data());
+
+  // Register rec1 with 0 EOS as min_fee
+  bridgereg("rec1"_n, make_asset(0));
+
+  // Fund evm1 address with 100 EOS
+  evm_eoa evm1;
+  const int64_t to_bridge = 1000000;
+  transfer_token("alice"_n, "evm"_n, make_asset(to_bridge), evm1.address_0x());
+
+  // Deploy emiter contract and get its address and account ID
+  auto contract_addr = deploy_contract(evm1, evmc::from_hex(emiter_bytecode).value());
+  uint64_t contract_account_id = find_account_by_address(contract_addr).value().id;
+  produce_blocks(1);
+
+  // Call method "start" on emiter contract (sha3('start(string,uint)') = 0xa5cc93e4)
```

Comment:
Please update comment to:
```
// Call method "go" on emitter contract (sha3('go(string,uint256)') = 0xa5cc93e4)
```

Also update the comment below to use `go` instead of `start`.